### ISSUE TITLE
Serverless Offline not working

### ISSUE THREAD

**Issue by d3netxer** [serverless/serverless#8352]
Congrats on Serverless V2. Serverless Offline is not working for me: ``` sls offline```

outputs:
```
Serverless: Offline requires Serverless v1.x.x but found 2.4.0. Exiting.
```

How can I fix this?

**Comment by pratik-vii** [comment]
@d3netxer Hello! 
I have tested it and works for me with,
* serverless env details:
```
Node Version:              14.10.1
Framework Version:         2.4.0
Plugin Version:            4.0.4
SDK Version:               2.3.2
Components Version:        3.2.1
```
* serverless-offline@6.8.0

Try with these configurations.

**Comment by d3netxer** [comment]
Thanks for checking @pratik-vii! I think maybe my serverless-offline version got stuck at ver 3 for some reason.

Also I found out that If you have the Serverless Framework CLI installed locally within your Service (i.e. project) folder, that version of the Framework will be run, instead of a globally installed version.

Therefore I installed Serverless V1 locally: ``` npm install serverless@1.83 --save-dev```

and serverless-offline worked again too.

However, it is probably best to updated both serverless and serverless-offline to the latest versions.
